sample-workspace/TestContract/TestContract.csproj

```diff
@@ -5,7 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo.SmartContract.Framework" Version="2.9.3.1" />
+    <!-- <PackageReference Include="Neo.SmartContract.Framework" Version="2.9.3.1" /> -->
+    <Reference Include="$(HOME)/neo-devpack-dotnet/Neo.SmartContract.Framework/bin/Debug/netstandard1.6/Neo.SmartContract.Framework.dll" />

```

Comment:
Maybe you should just include the latest Neo.SmartContract.Framework.dll in this repo so other devs who checkout the code can run thru this test scenario?

Reply:
Could do, that's not quite enough though... You also need to fiddle with dotnet tools so the neon-de executable that gets used adds the metadata node to the ABI file. Might be better to just check in the AVM and ABI?

Reply:
works for me, though you should use the latest neon to build rather than neon-de. You'll need to change the DIrectory.build.targets file as well